INN/umbrella-caribbean
# Category archive

## Changes

This pull request makes the following changes:

- Pulls in the Largo `category.php` template and modifies it to remove the `largo_before_category_river` action
- Adds styling for the category posts to match posts displayed on the homepage

Desktop:
![screencapture-caribbean-test-2020-02-14-13_45_59](https://user-images.githubusercontent.com/18353636/74559222-dd1eda80-4f31-11ea-94ff-766ad523e2ca.png)

Mobile:
![screencapture-caribbean-test-2020-02-14-13_54_45](https://user-images.githubusercontent.com/18353636/74559210-d8f2bd00-4f31-11ea-8599-2c63f951409c.png)

## Why



For #88

## Testing/Questions

Features that this PR affects:

- Category archives


Questions that need to be answered before merging:

- [ ] @MirandaEcho Does this format look ok or do we need to do anything else?

Steps to test this PR:

1. Add a sidebar to a category
2. View it on the frontend on desktop and mobile and make sure it looks ok
